packages/experiments/src/components/Chiclet/ChicletXsmall.types.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react';
+import { ChicletXsmallBase } from './ChicletXsmall.base';
+import { ITheme, IStyle } from '../../Styling';
+import { IStyleFunctionOrObject, IRefObject } from '../../Utilities';
+
+export interface IChicletXsmall {}
+
+export interface IChicletXsmallStyles {
+  /**
+   * Style for the root element.
+   */
+  root?: IStyle;
```

Comment:
Shouldn't all style regions be required props? I know there is inconsistency among OUFR controls but last time I checked we were supposed to make them required.